node/src/command.rs

```diff
@@ -394,7 +363,7 @@ pub fn run() -> Result<()> {
 
 				let genesis_state = with_runtime_or_err!(chain_spec, {
 					{
-						let block: Block = generate_genesis_block(&chain_spec, state_version)?;
+						let block: Block = generate_genesis_block(&**chain_spec, state_version)?;
```

Comment:
chain_spec is `& Box...` and we need the to get the thing in the Box. thus the double